src/NuGet.Core/NuGet.Configuration/Resources.resx

```diff
@@ -309,4 +309,7 @@
 1 - config file path
 </comment>
   </data>
+  <data name="ShowError_UnknownOrigin" xml:space="preserve">
+    <value>File location not known</value>
```

Comment:
Should this end in a period?
```suggestion
    <value>File location not known.</value>
```

Reply:
No, I don't think it. It gets used inside another message: https://github.com/NuGet/NuGet.Client/blob/5502baa16a294bb1290b6fd2a135694ed2884a59/src/NuGet.Core/NuGet.Configuration/Resources.resx#L307

In theory this should never be shown to customers, but because of how `SettingsBase` is implemented, `.Origin` has to be marked as nullable because of an obscure design choice, but in practise when saving changes to nuget.config files, it will always be not null.